profiles/learner-profiles.md

```diff
@@ -1,5 +1,15 @@
 ---
-title: FIXME
+title: Learner profiles
 ---
 
-This is a placeholder file. Please add content here.
+**Ana Fátima Fernandes dos Anjos Ferreira** is a biologist working on the effect of chemicals in aquatic environments. She works most of the time with R. R was never part of her training in the biology school, but she compensated for that with self-study, tutorials, both written and in Youtube. She did this self-learning mostly during weekends and free-time.  
+
+Eventually she became the acclaimed R authority in her lab. Since then, she has developed several scripts for her colleagues. This is a double-edged sword, because sometimes her colleagues underestimate the effort this takes. Nor does her supervisor fully appreciate the work it takes to write reusable R code. She finds it hard to get support; she has some friends she contacts with questions about R, but there is no one in her direct professional environment who can take the time to work deeply through an issue and spend time on her code to improve it.
+
+Often, she sees herself struggling with coding problems. That’s ok, just life, but she has this annoying feeling... this voice in the back of her head that says: “There should be an easier way for doing this...”. As a postdoc she is incredibly busy, and finds it hard to find time to get back to improving her old projects, although she knows she probably should.
+
+**Mzkizi Johnson** is an environmental scientist who is used to working with Python. He’s getting involved in a new project that has a lot of R and he needs to learn the ins and outs of it (best practices, etc.).
+
+Mzkizi was born in South Africa but has been living in Germany ever since his under-graduates. He likes cricket and has to do with watching streams, since the level of cricket competition is abismal in Germany.
+
+**Teun de Graaf** is a middle-aged statistician who has a permanent position as research support at the department of veterinary sciences. He is often called to help students and PhD candidates with their R scripts. He enjoys wrangling their data, mostly applying his expertise to make sure they choose the right tests for their question. He is self-taught in R and works mostly on a metaphorical island with students who rarely know anything about R. He does not have direct colleagues with whom he can discuss R with. He does know that there is more to R than scripting analyses, but rarely gets to develop this knowledge further, as the questions he gets from students are mostly straightforward and require a simple script to answer. However, he is interested in improving his workflow, and has the gnawing feeling he is often repeating himself when he writes yet another script for yet another student.
```

Comment:
```suggestion
**Teun de Graaf** is a middle-aged statistician who has a permanent position as research support at the department of veterinary sciences.
He is often called to help students and PhD candidates with their R scripts.
He enjoys wrangling their data, mostly applying his expertise to make sure they choose the right tests for their question.
He is self-taught in R and works mostly on a metaphorical island with students and researchers who rarely know anything about R.
He does not have direct colleagues with whom he can discuss R.
He does know that there is more to R than scripting analyses, but rarely gets to develop this knowledge further, as the questions he gets are mostly straightforward and require a simple script to answer.
However, he is interested in improving his workflow, and has the gnawing feeling he is often repeating himself when he writes yet another script for yet another student.
```

New lines in a markdown allows more precise tracking of changes. There was also a small language error ("with whom he can discuss R with"), and I tweaked the references to "students" a bit.